platform/security/sts/security-sts-usernametokenvalidator/src/main/java/org/codice/ddf/security/validator/username/UsernameTokenValidator.java

```diff
@@ -276,28 +227,6 @@ public TokenValidatorResponse validateToken(TokenValidatorParameters tokenParame
             String tokenRealm = null;
             if (usernameTokenRealmCodec != null) {
```

Comment:
We no longer use the `usernameTokenRealmCodec` and it is not injected/always null. It should be removed from this file and this block of code removed.